web/modules/custom/foreningsmentor/foreningsmentor.module

```diff
@@ -423,3 +407,26 @@ function foreningsmentor_views_pre_render(ViewExecutable $view) {
     };
   }
 }
+
+/**
+ * Implements hook_mail
+ *
+ * Change mail header and body.
+ *
+ * @param $key
+ * @param $message
+ * @param $params
+ * @throws \Twig\Error\RuntimeError
+ */
+function foreningsmentor_mail($key, &$message, $params) {
+  if ('signup' === $key) {
```

Comment:
Are we hooking into a generic Drupal signup processes here? Otherwise, I think a more specific mail key, `foreningsmentor_signup`, say, should be used.

Reply:
Changed